tests/krb5-krb5_msg_type/test.yaml

```diff
@@ -0,0 +1,41 @@
+# *** Add configuration here ***
+requires:
+  features:
```

Comment:
I think we will want min-version: 7

Reply:
Done in https://github.com/OISF/suricata-verify/pull/572/commits/1a993956803a8b72b42ad6631f9db78cc95c413b